.eslintrc

```diff
@@ -2,5 +2,9 @@
   "extends": "plugin:shopify/esnext",
   "env": {
     "browser": true
+  },
+  "rules": {
+  	"promise/always-return": 0,
```

Comment:
I figure we don't need those rules in this project anyway, so I'll just leave them disabled globally